frontend/packages/ui/src/Layouts/PageLayout.tsx

```diff
@@ -241,26 +241,18 @@ const NAV_WIDTH = 300
 const CONTENT_MAX_WIDTH = 1100
 
 export const PageLayout = styled(Page)`
-  display: grid;
   position: relative;
+  display: flex;
+  flex-direction: column;
   min-height: 100vh;
   z-index: 0;
-  grid-template-columns: 0 100%;
-  grid-template-rows: 0 100%;
   background: ${Colors.background};
   will-change: grid-template-columns;
-  transition: grid-template-columns 0.3s;
-  ${props =>
-    props.header != null &&
-    `
-      grid-template-rows: ${HEADER_HEIGHT}px calc(100% - ${HEADER_HEIGHT}px);
-  `}
+  transition: grid-template-columns 0.3s;  
```

Comment:
remove this will-change and transition
and remove navigation, it seams to be working fine with display: flex;